x-pack/plugins/ingest_manager/server/services/epm/elasticsearch/transform/install.ts

```diff
@@ -33,89 +33,100 @@ export const installTransformForDataset = async (
   registryPackage: RegistryPackage,
   paths: string[],
   callCluster: CallESAsCurrentUser,
-  savedObjectsClient: SavedObjectsClientContract
+  savedObjectsClient: SavedObjectsClientContract,
+  logger: Logger
```

Comment:
Our current pattern is to have functions use the logger like `const logger = appContextService.getLogger();` vs adding them to every function signature. 

e.g. https://github.com/elastic/kibana/blob/46a268fb99606d29f7bd3b27c454cc210f49f9b9/x-pack/plugins/ingest_manager/server/services/epm/elasticsearch/ingest_pipeline/remove.ts#L19